Scenes/Tavern/TavernScreen.tscn

```diff
@@ -116,51 +117,58 @@ texture_pressed = null
 button_label = ""
 screen_name = "PublicanScreen"
 
-[node name="RoomScreen" parent="." instance=ExtResource( 5 )]
+[node name="RoomScreen" parent="." instance=ExtResource( 8 )]
 visible = false
 
-[node name="WizardScreen" parent="." instance=ExtResource( 6 )]
+[node name="WizardScreen" parent="." instance=ExtResource( 9 )]
 visible = false
 
-[node name="TradesmanScreen" parent="." instance=ExtResource( 7 )]
+[node name="TradesmanScreen" parent="." instance=ExtResource( 10 )]
 visible = false
 
-[node name="PublicanScreen" parent="." instance=ExtResource( 8 )]
+[node name="PublicanScreen" parent="." instance=ExtResource( 11 )]
 visible = false
 
 [node name="Resources" type="Control" parent="."]
 visible = false
 margin_right = 40.0
 margin_bottom = 40.0
-script = ExtResource( 9 )
+script = ExtResource( 12 )
 
 [node name="MarginContainer" type="MarginContainer" parent="Resources"]
 anchor_right = 1.0
 anchor_bottom = 1.0
+margin_right = 47.0
```

Comment:
I think that for MarginContainer you should use `custom_constants/margin_*` instead of `margin_*` . [Read more](https://docs.godotengine.org/en/3.2/classes/class_margincontainer.html)

Reply:
Right